assets/base/i18n/en.json

```diff
@@ -1537,6 +1537,7 @@
   "integrations.outgoingWebhook.title": "Outgoing Webhook",
   "integrations.successful": "Setup Successful",
   "intro_messages.DM": "This is the start of your direct message history with {teammate}.<br />Direct messages and files shared here are not shown to people outside this area.",
+  "intro_messages.GM": "This is the very beginning of your group message.",
```

Comment:
this one we should remove it from here, we should be adding only those that have a `mobile.` prefix.